package.json

```
@@ -168,7 +168,7 @@
     "atom-package-deps": "5.1.0",
     "babel-preset-env": "1.7.0",
     "consistent-path": "2.0.3",
-    "crypto-random-string": "2.0.0",
+    "crypto-random-string": "^3.2.0",
```

```suggestion
    "crypto-random-string": "3.2.0",
```
renovate will give us PRs when there are new versions, otherwise with the lockfile we would be stuck with the old version from when this PR was submitted until the range broke.